scripts/geopmpy/launcher.py

```diff
@@ -918,25 +920,15 @@ def get_idle_nodes(self):
         Returns a list of the names of compute nodes that are currently
         available to run jobs.
         """
-        if self._is_slurm_enabled:
```

Comment:
i'm not sure i understand this change.  I think the original idea is that you may have Slurm as a *resource manager* but use, say, mpiexec from Intel MPI to do *application launch*.  In this way it does make sense to have logic about Slurm in the base class.